tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserList.java

```diff
@@ -451,6 +451,7 @@ private void putShownUser(JSONArray shownUsers, UserSettings u) throws JSONExcep
         boolean havePermToChange = me.isAdminFor(user);
         boolean userCanDeleteUser = UserRegistry.userCanDeleteUser(me, user.id, user.userlevel);
         VoteResolver.Level level = VoteResolver.Level.fromSTLevel(user.userlevel);
+        // TODO: why doesn't this use u.toJSONObject()?
```

Comment:
"TODO" comment should include a ticket number

Reply:
good. i'll update this in a followon